app/api_topologies.go

```diff
@@ -21,6 +21,14 @@ var (
 	topologyRegistry = &registry{
 		items: map[string]APITopologyDesc{},
 	}
+	pseudoFilter = APITopologyOptionGroup{
+		ID:      "pseudo",
+		Default: "show",
+		Options: []APITopologyOption{
+			{"show", "Show pseudo nodes", nil, false},
+			{"hide", "Hide pseudo nodes", render.IsNotPseudo, true},
```

Comment:
While perfectly valid as an internal name, I am not sure _pseudo_ will tell much to the end user. Maybe _Uncontained_ if it's accurate enough?